documentation/largescaledeployment.md

```diff
@@ -1,13 +1,13 @@
 ---
 title: Recommended configuration for large scale cluster deployment
 ---
-vSphere Cloud Provider is successfully validated with 500 nodes Kubernetes cluster.
+vSphere Cloud Provider is successfully validated with 1000 nodes Kubernetes cluster.
 
-Below is the configuration recommended for 500 nodes Kubernetes cluster.
+Below is the configuration recommended for 1000 nodes Kubernetes cluster.
```

Comment:
We need to change this wording.
We are qualified to "recommend" this configuration. What is this recommended for - test/dev, production, which workload types?
Let's reword this as follows -  "The following configuration is used to validate the scale limits for VCP.